common/changes/@gooddata/sdk-ui-all/ine-one-4890_2021-02-24-15-53.json

```diff
@@ -0,0 +1,11 @@
+{
+    "changes": [
+        {
+            "packageName": "@gooddata/sdk-ui-all",
+            "comment": "Remove `removeDateItems` flag in loadDateDataSets in api-client-bear. Now date bucket items are always removed from request loadDateDataset, but with \"multiple date datasets\" feature we need to keep some of them to get valid available date datasets and avoid execution error with incompatible date dimensions.",
```

Comment:
This is not correct, now the date items are never removed in the SDK, right? Also, please write this from the user's perspective. How it affects them? Should they do anything?